### DISCUSSION TITLE
How does the deployment process work in Theia?

### DISCUSSION THREAD

**Discussion by sboudreaux96** [eclipse-theia/theia#9481] [Q&A]
Hello - I am trying to figure out how the process works and what application is used to host it?

**Comment by vince-fugnitto** [comment]
> Hello - I am trying to figure out how the process works and what application is used to host it?

@sboudreaux96 thank you for the discussion, deployement is left up to you the application developer, and you are free to choose any deployment you want for either `electron` or `browser` targets.

We have the following [theia-apps](https://github.com/theia-ide/theia-apps) repository with contains examples on how electron and browser apps can be distributed:
- browser as docker images
- electron as a packaged electron application (with electron-builder)
